src/components/Cart/Cart.js

```diff
@@ -14,7 +15,7 @@ import CartItemsList from '../CartItemsList';
 import ShippingDetails from '../ShippingDetails';
 import TextLink from '../TextLink';
 
-function Cart({ items, refillDemoCart }) {
+function Cart({ items, refillDemoCart = noop }) {
```

Comment:
Not sure if **noop**  is really good idea here.
I would say for this component to function properly this handler is actually required.
If we wan to make it optional I would say `{refillDemoCart && isCartEmpty && <TextLink />}` (line 34) is much better solution since it hides whole trigger.
And you can always pass noop in test to satisfy prop check.